src/components/timeout-dialog/timeout-dialog.js

```diff
@@ -255,7 +259,7 @@ function TimeoutDialog ($module) {
       var counter = getSecondsRemaining()
       updateCountdown(counter, $countdownElement)
       if (counter <= 0) {
-        signOut()
```

Comment:
Looks like the signout function isn't used anymore. Can it be removed?

Reply:
`signOut()` is used for for the link and `timeout()` for the timeout, although since they aren't *re-used* I could just inline the Redirect call?